ddsrecorder_yaml/src/cpp/recorder/YamlReader.cpp

```diff
@@ -30,6 +31,14 @@ YamlReader::get<mcap::McapWriterOptions>(
         const Yaml& yml,
         const YamlReaderVersion version)
 {
+    const std::set<TagType> tags = {
```

Comment:
Same comment as in the Pipe, I would save all the sets in ``yaml_configuration_tags.hpp``

Reply:
Same reply as in the DDS Pipe: https://github.com/eProsima/DDS-Pipe/pull/85#discussion_r1559556618